reference/payment_processing.yml

```diff
@@ -317,6 +317,7 @@ paths:
                   - status
                   - title
                   - type
+      security: []
```

Comment:
shouldn't this diff show something removed? or is this empty array overriding a security param at a higher level?

and what about the `Authorization: PAT {{pat_token}}` header? does that not count as security?

Reply:
let's let it go for now. Make a low-priority ticket to follow up with payments.

Reply:
Resolving conversation, see [DEVDOCS-4003](https://bigcommercecloud.atlassian.net/browse/DEVDOCS-4003)